cloudant-client/src/test/java/com/cloudant/tests/HttpProxyTest.java

```diff
@@ -217,6 +225,8 @@ public void resetAuthenticator() {
         if (useProxyAuth && useHttpsServer && !okUsable) {
             Authenticator.setDefault(null);
         }
+        // Reset the disabled schemes property
+        System.setProperty("jdk.http.auth.tunneling.disabledSchemes", defaultDisabledList);
```

Comment:
Good spot, I've fixed it up in 30a24ac.
Basically my reset was happening outside of the if block, thus setting the value to `""` which had the effect of setting it for the test that needed it.
Unfortunately it turns out the property is unset for the default so we can't read the value to reset it anyway.